common4j/src/main/com/microsoft/identity/common/java/util/ported/LocalBroadcaster.java

```diff
@@ -26,13 +26,16 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 import lombok.NonNull;
 
 public enum  LocalBroadcaster {
     INSTANCE;
 
     private static final String TAG = LocalBroadcaster.class.getSimpleName();
+    private static final ExecutorService sBroadcastExecutor = Executors.newSingleThreadExecutor();
 
```

Comment:
Should the register/unregister function also utilize this executor?

Reply:
Or maybe switch mReceivers to concurrentHashMap?

Reply:
switched to ConcurrentHashMap